Formula/s/sby.rb

```diff
@@ -0,0 +1,57 @@
+class Sby < Formula
+  include Language::Python::Shebang
+
+  desc "Front-end for Yosys-based formal verification flows"
+  homepage "https://symbiyosys.readthedocs.io/en/latest/"
+  url "https://github.com/YosysHQ/sby/archive/refs/tags/v0.49.tar.gz"
+  sha256 "469e3542569e43de454643e4c5f3e51ca6780b65bf5c0798a64f695ed4b12f7d"
+  license "ISC"
+  head "https://github.com/YosysHQ/sby.git", branch: "main"
+
+  depends_on "python@3.13"
+  depends_on "yices2"
```

Comment:
Oh... I missed this point.

Reply:
Made it an test dependency only and updated associated PR https://github.com/Homebrew/homebrew-core/pull/207218.